### ISSUE TITLE
Missing type for DataforseoLabsGoogleKeywordIdeasLiveItem

### ISSUE THREAD

**Issue by BenJackGill** [dataforseo/TypeScriptClient#6]
We have types such as this for the Labs historical data `DataforseoLabsGoogleHistoricalSearchVolumeLiveItem`. 

But there is no similar type for the Keyword Ideas endpoint. We are missing a type such as `DataforseoLabsGoogleKeywordIdeasLiveItem`.

**Comment by lVics** [comment]
Not every endpoint has its own unique model name in the response object. It depends on the similarity of the structure with other endpoints. The `items` field in `DataforseoLabsGoogleKeywordIdeasLiveResultInfo` has a structure that we called `KeywordDataInfo` because there are 9 endpoints that have the same structure: Domain Intersection, Keyword Ideas, Keyword Suggestions, Category Keywords, Site Keywords, Page Intersection, Ranked Keywords, Related Keywords, and Top Searches.
To avoid duplication of objects, we have combined them into one class
